src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,42 @@
+package duke.commands;
+
+import duke.exceptions.DukeException;
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.Ui;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Represents abstract class for all user command
+ */
+public abstract class Command {
+    protected List<String> userInputList;
+
+    /**
+     * Creates command from userInput in String
+     *
+     * @param userInput User input in string formats
+     * @throws DukeException If user doesn't specify description/time of task
+     */
```

Comment:
Remember to add punctuations behind each parameter description and the header comments.